src/base/random.hh

```diff
@@ -102,18 +101,20 @@ class Random : public Serializable
     random()
     {
         // [0, 1) for real types
-        std::uniform_real_distribution<T> dist;
-        return dist(gen);
+        return (T) random<uint64_t>() /
```

Comment:
There are limits to this implementation. I.e., the resulting distribution is not perfectly uniform. But I think that's OK in our use case.

Reply:
That's correct, but also true for the current code. Maybe I can add a `warn` statement.

Reply:
Done

Reply:
Sorry for not being clear... I was just making a generic comment, not asking you to make any changes. I would vote against having a `warn` statement here so that gem5 is less verbose. I feel like a lot of our warnings are ignored because we have so many already. I would suggest adding a comment. If you think a warning is more appropriate, then it should be `warn_once`.